test/mocha.opts

```diff
@@ -1,6 +1,6 @@
 --ui tdd
 --require test/helper
---timeout 240s
+--timeout 480s
```

Comment:
because [this test](https://github.com/taskcluster/docker-worker/pull/177/files#diff-0f281f5ca74802b8893be95ed5839938R87) takes some time because of the big files.